docs/components/Typography/index.ts

```diff
@@ -1,3 +1,5 @@
+import TextStylesPage from './TextStylesPage';
 import TypesPage from './types-page';
 
 export default TypesPage;

```

Comment:
It is my error, please help me put the TypesPage also in the export like`export { TextStylesPage, TypesPage}` and change the import in the file typography-types-page.stories.mdx like `import { TypesPage } from './components/Typography';